cookbooks/vankeken_subduction/vankeken_corner_flow.prm

```diff
@@ -0,0 +1,163 @@
+set Pressure normalization                 = no
+set Surface pressure                       = 0
+set Use conduction timestep                = true
+set End time                               = 3.4e8
+set Resume computation                     = false
+
+# Turn prescribed velocities on, this is necessary for the subducting
+# and overriding plates.
+set Prescribe internal velocities = true
+
+# Solver parameters
+subsection Solver parameters
+  subsection Stokes solver parameters
+    set Stokes solver type                              = block AMG
+    set Maximum number of expensive Stokes solver steps = 1000
+  end
+end
+
+# q2q1 discretization
+subsection Discretization
+  set Stokes velocity polynomial degree = 2
+  set Use locally conservative discretization = false
+end
+
+# Incompressible
+subsection Formulation
+  set Formulation = Boussinesq approximation
+end
+
+# Use the specialized geometry model for this cookbook
+subsection Geometry model
+  set Model name = vanKeken box
+end
+
+# Checkpoint every 200 steps
+subsection Checkpointing
+  set Steps between checkpoint = 200
+end
+
+# Prescribe a half space cooling model on the left boundary and in the overriding plate
+subsection Initial temperature model
+  set Model name = function
+  subsection Function
+    set Variable names = x,y,t
+    # H is the model Y extent, Hl is the overriding plate thickness, TS is the surface temperature
+    # T0 is the temperature at the base of the slab
+    set Function constants = H=6.0e5, Hl=5.0e4, TS=273.0, T0=1573.0, kappa=0.7272e-6, t50=1.5778463e15
+    set Function expression = if(x<=(H-y),TS+(T0-TS)*(1.0-erfc( (H-y)/(2.0*sqrt(kappa*t50))) ), \
+                 if(x>(H-y) & y>(H-Hl), TS+(T0-TS)*(H-y)/Hl, \
+                    T0 ) )
+  end
+end
+
+# Bottom and right boundary is open
+subsection Boundary traction model
+  set Prescribed traction boundary indicators = bottom:zero traction
+end
+
+# Keep the temperature fixed on the left, top and right boundaries
+subsection Boundary temperature model
+  set List of model names                           = initial temperature
+  set Allow fixed temperature on outflow boundaries = false
+  set Fixed temperature boundary indicators         = top, left, right
+end
+
+# Prescribe a velocity of 5 cm/yr on the left boundary, 0 on the top
+subsection Boundary velocity model
+  set Prescribed velocity boundary indicators = left: function, top:function
+  subsection Function
+    set Variable names = x,y,t
+    set Function constants = H=6.01e5, v=0.035355339
+    set Function expression = if( x<=(H-y), v, 0.0) ; \
+                              if( x<=(H-y),-v, 0.0 )
+  end
+end
+
+# No gravity
+subsection Gravity model
+  set Model name = vertical
+  subsection Vertical
+    set Magnitude = 0.
+  end
+end
+
+# Use the visco plastic material model and specify and isoviscous material with
+# a viscosiy of 1e20 Pa s
+subsection Material model
+  set Model name         = visco plastic
```

Comment:
For my own curiosity, is that the material model used in the paper? I thought it was really just a simple viscosity model. Or perhaps there was an earlier paper where that was the case, and the 2008 one used a more complex viscosity?

Reply:
You are correct, I essentially made this a simple viscosity model by forcing a constant viscosity of 1e20, but this could much more easily have been done using the simpler model

Reply:
I'm not opposed to sticking with the setup you have here, but it would be nice to add a comment that the material really is just viscous and that the choice of material model used here was out of convenience, and that it would be easy to switch to a simpler material model.